script.js

```diff
@@ -61,25 +60,26 @@ colorApp.getColors = (userChoice) => {
             } 
         })
         .then((jsonData) => {
-            colorApp.displayColors(jsonData);
+            console.log(jsonData)
+            // colorApp.displayColors(jsonData);
         })
         .catch((errorResponse) => {
             alert(errorResponse);
         })
 }
 
-colorApp.displayColors = (arrayOfColors) => {
-    const colorHex0 = arrayOfColors[0].hex;
-    document.querySelector('.color0').style.backgroundColor = colorHex0;
-    const colorHex1 = arrayOfColors[1].hex;
-    document.querySelector('.color1').style.backgroundColor = colorHex1;
-    const colorHex2 = arrayOfColors[2].hex;
-    document.querySelector('.color2').style.backgroundColor = colorHex2;
-    const colorHex3 = arrayOfColors[3].hex;
-    document.querySelector('.color3').style.backgroundColor = colorHex3;
-    const colorHex4 = arrayOfColors[4].hex;
-    document.querySelector('.color4').style.backgroundColor = colorHex4;    
+// colorApp.displayColors = (arrayOfColors) => {
```

Comment:
```suggestion
colorApp.displayColors = (arrayOfColors) => {
const colorHex0 = arrayOfColors['colors'][0]['hex']['value'];
    document.querySelector('.color0').style.backgroundColor = colorHex0;
    const colorHex1 = arrayOfColors['colors'][1]['hex']['value'];
    document.querySelector('.color1').style.backgroundColor = colorHex1;
    const colorHex2 = arrayOfColors['colors'][2]['hex']['value'];
    document.querySelector('.color2').style.backgroundColor = colorHex2;
    const colorHex3 = arrayOfColors['colors'][3]['hex']['value'];
    document.querySelector('.color3').style.backgroundColor = colorHex3;
    const colorHex4 = arrayOfColors['colors'][4]['hex']['value'];
    document.querySelector('.color4').style.backgroundColor = colorHex4;
    document.querySelector('header').style.background = `linear-gradient(135deg, white, ${colorHex0}, ${colorHex1}, ${colorHex2})`;
 }
```
Used bracket notation to crawl through the object and get to the hex code for each color in the array. Probably a cleaner way to do this with destructuring? We can refactor once we make sure this works at baseline!?

Reply:
love that idea, let's try destructuring  later on :)